DemoApp/ios/Podfile

```diff
@@ -9,10 +9,12 @@ target 'DemoApp' do
   # use_frameworks!
 
   # Pods for DemoApp
-  pod 'AppCenter/Push', '~> 1.14.0'
-  pod 'AppCenter/Crashes', '~> 1.14.0'
-  pod 'AppCenter/Analytics', '~> 1.14.0'
-  pod 'AppCenterReactNativeShared', '~> 1.13.0'
+  pod 'AppCenter/Push', '~> 2.0.1'
+  pod 'AppCenter/Crashes', '~> 2.0.1'
+  pod 'AppCenter/Analytics', '~> 2.0.1'
+  pod 'AppCenterReactNativeShared', '~> 2.0.0'
 
   platform :ios, '9.0'
+  pod 'RNGestureHandler', :path => '../node_modules/react-native-gesture-handler'
```

Comment:
please do the same comment trick we did in TestApp. This dependency is optional. Running react-native link once will add this one. If you react native link again, it will try to link React via Pods and this will cause hundreds of files to be committed next time.

Please comment out and copy the comment that we have in TestApp.